src/dynamic_importer/main.py

```diff
@@ -251,6 +256,13 @@ def walk_directories(config_dir, file_types, output_dir):
             if dir in dirs:
                 dirs.remove(dir)
 
+        # skip over user-specified non-config directories
+        for dir in exclude_dirs:
```

Comment:
Does this still exclude your default list too?

Reply:
I left that loop above the new code. But you've given me a better idea of how to concatenate the two lists.

Reply:
Eh, nevermind. The logic for finding referenced directories is too different from the existing logic. It will end up getting refactored later.